fuzz/agnes-fuzz.c

```diff
@@ -0,0 +1,22 @@
+#include <stdint.h>
```

Comment:
Something to keep in mind is that you only really need LLVMFuzzerInitialize if you plan on modifying or reading argv/c.